modules/tinymce/src/plugins/autoresize/test/ts/browser/AutoresizePluginTest.ts

```diff
@@ -53,130 +36,191 @@ describe('browser.tinymce.plugins.autoresize.AutoresizePluginTest', () => {
+      editor.execCommand('FontSize', false, '20pt');
+      assertScrollPositionGreaterThan(window, 3500);
+    });
+
+    it('TINY-7291: Editor does not scroll to the top on undo/redo (SetContent & NodeChange trigger)', () => {
+      const editor = hook.editor();
+      editor.resetContent('<div style="height: 5000px;">a</div><p>Some content</p>');
+      window.scrollTo(0, 5000);
+      TinySelections.setCursor(editor, [ 1, 0 ], 12);
+      TinyContentActions.type(editor, '. More content...');
+      editor.undoManager.add();
+      assertScrollPositionGreaterThan(window, 3500);
+      editor.execCommand('Undo');
+      assertScrollPositionGreaterThan(window, 3500);
+      editor.execCommand('Redo');
+      assertScrollPositionGreaterThan(window, 3500);
+    });
   });
 
-  it('TBA: Editor size increase with async loaded content', async () => {
-    const editor = hook.editor();
-    // Note: Use a min-height here to account for different browsers rendering broken images differently
-    editor.setContent('<div style="min-height: 35px;"><img src="#" /></div><div style="height: 5500px;"></div>');
-    await Waiter.pTryUntil('wait for editor content height', () => assertEditorContentApproxHeight(editor, 5585), 10, 3000);
-    // Update the img element to load an image
-    const image = editor.dom.select('img')[0];
-    editor.dom.setAttrib(image, 'src', 'http://moxiecode.cachefly.net/tinymce/v9/images/logo.png');
-    // Content height + div image height (84px) + bottom margin = 5634
-    await Waiter.pTryUntil('wait for editor content height', () => assertEditorContentApproxHeight(editor, 5634), 10, 3000);
-    await Waiter.pTryUntil('wait for editor height', () => assertEditorHeightAbove(editor, 5634), 10, 3000);
+  context('TINY-9123', () => {
+    const resizeEventsCount = Cell(0);
+    const hook = TinyHooks.bddSetup<Editor>({
+      plugins: 'autoresize fullscreen',
+      menubar: false,
+      toolbar: 'autoresize',
+      base_url: '/project/tinymce/js/tinymce',
+      content_style: 'html { min-height: 100%; } body { max-width: 820px; margin: 10px auto 0; min-height: calc(100vh - 10px) }',
+      setup: (editor: Editor) => {
+        editor.on('ResizeEditor', () => {
+          resizeEventsCount.set(resizeEventsCount.get() + 1);
+        });
+      }
+    }, [ AutoresizePlugin, FullscreenPlugin ], true);
+
+    beforeEach(() => {
+      resizeEventsCount.set(0);
+    });
+
+    it('TINY-9123: it should not continue to resaize in some specific condition', async () => {
+      const editor = hook.editor();
+      editor.setContent('<div style="height: 250px;">a</div>');
+      await Waiter.pWait(2000);
+      assert.isAtMost(resizeEventsCount.get(), 10, 'Should have fired a ResizeEditor event at most 10 time');
+    });
+
+    // this is not working because of a bug in the `setContent`
+    it.skip('TINY-9123: it should continue to resaize if the content expands or contract', async () => {
```

Comment:
since the test is not working because of a bug in `setContent` my idea was to preserve the test and remove the skip when that bug will be fixed.

Should I open the jira and then write the code in the comment?

Reply:
Ok, I see now. Yeah I think adding a TODO comment that mentions a Jira ticket would be helpful.

Reply:
ok, my bad there were no bug in `setContent` I just forgot to remove the fixed height in the container